{"kind":"symbol","schemaVersion":{"patch":0,"major":0,"minor":3},"abstract":[{"type":"text","text":"An argument supplied to assert security privilege was not a host"},{"type":"text","text":" "},{"type":"text","text":"security port."}],"hierarchy":{"paths":[["doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX","doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/MachError"]]},"metadata":{"extendedModule":"Foundation","platforms":[{"name":"iOS","deprecated":false,"introducedAt":"8.0","beta":false,"unavailable":false},{"introducedAt":"10.10","deprecated":false,"unavailable":false,"beta":false,"name":"macOS"},{"name":"tvOS","deprecated":false,"introducedAt":"9.0","unavailable":false,"beta":false},{"unavailable":false,"name":"watchOS","beta":false,"deprecated":false,"introducedAt":"2.0"}],"roleHeading":"Type Property","title":"invalidSecurity","externalID":"s:10Foundation9MachErrorV15invalidSecurity6Darwin0bC4CodeOvpZ","symbolKind":"property","role":"symbol","modules":[{"relatedModules":["Foundation"],"name":"FoundationX"}],"fragments":[{"text":"static","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"invalidSecurity","kind":"identifier"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"MachError","preciseIdentifier":"s:10Foundation9MachErrorV"},{"kind":"text","text":"."},{"text":"Code","kind":"typeIdentifier","preciseIdentifier":"s:10Foundation9MachErrorV4Codea"}]},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/MachError\/invalidSecurity"},"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["macOS"],"languages":["swift"],"tokens":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"invalidSecurity","kind":"identifier"},{"kind":"text","text":": "},{"identifier":"doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/MachError","preciseIdentifier":"s:10Foundation9MachErrorV","text":"MachError","kind":"typeIdentifier"},{"text":".","kind":"text"},{"text":"Code","kind":"typeIdentifier","preciseIdentifier":"s:10Foundation9MachErrorV4Codea","identifier":"doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/MachError\/Code"},{"kind":"text","text":" { "},{"text":"get","kind":"keyword"},{"text":" }","kind":"text"}]}]}],"sections":[],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/foundationx\/macherror\/invalidsecurity"]}],"references":{"doc://com.swallow.documentation.FoundationX/documentation/FoundationX/MachError":{"fragments":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"MachError"}],"identifier":"doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/MachError","title":"MachError","role":"symbol","abstract":[{"type":"text","text":"Describes an error in the Mach error domain."}],"url":"\/documentation\/foundationx\/macherror","type":"topic","kind":"symbol","navigatorTitle":[{"kind":"identifier","text":"MachError"}]},"doc://com.swallow.documentation.FoundationX/documentation/FoundationX":{"abstract":[],"kind":"symbol","title":"FoundationX","identifier":"doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX","url":"\/documentation\/foundationx","role":"collection","type":"topic"},"doc://com.swallow.documentation.FoundationX/documentation/FoundationX/MachError/Code":{"fragments":[{"kind":"keyword","text":"typealias"},{"text":" ","kind":"text"},{"kind":"identifier","text":"Code"}],"identifier":"doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/MachError\/Code","title":"MachError.Code","role":"symbol","abstract":[],"url":"\/documentation\/foundationx\/macherror\/code","type":"topic","kind":"symbol","navigatorTitle":[{"kind":"identifier","text":"Code"}]},"doc://com.swallow.documentation.FoundationX/documentation/FoundationX/MachError/invalidSecurity":{"abstract":[{"text":"An argument supplied to assert security privilege was not a host","type":"text"},{"type":"text","text":" "},{"type":"text","text":"security port."}],"url":"\/documentation\/foundationx\/macherror\/invalidsecurity","role":"symbol","fragments":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"kind":"identifier","text":"invalidSecurity"},{"kind":"text","text":": "},{"preciseIdentifier":"s:10Foundation9MachErrorV","text":"MachError","kind":"typeIdentifier"},{"kind":"text","text":"."},{"text":"Code","kind":"typeIdentifier","preciseIdentifier":"s:10Foundation9MachErrorV4Codea"}],"kind":"symbol","type":"topic","identifier":"doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/MachError\/invalidSecurity","title":"invalidSecurity"}}}